### ISSUE TITLE
Add support for linear + batchnorm1d fusion

### ISSUE THREAD

**Issue by raghuramank100** [pytorch/pytorch#37383]
Add support for fusion of batch norm1d with linear layer. 

cc @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a

**Comment by vkuzo** [comment]
https://github.com/pytorch/pytorch/pull/50748 adds this
